posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -0,0 +1,353 @@
+
+You can now control whether the Request Timing feature collects thread dumps. By setting the **new** `enableThreadDumps` Request Timing server configuration attribute to false, thread dumps will not be created during hung requests. If the new server configuration attribute is set to true or not specified at all, thread dumps will still be created.
+   
+The new Request Timing server configuration attribute can be configured in your server.xml as follows:
+
+[source, xml]
+----
+<requestTiming includeContextInfo="true" slowRequestThreshold="120s" hungRequestThreshold="10s" sampleRate="1" enableThreadDumps="false"></requestTiming>`
+----
+
+
+The `enableThreadDumps` server configuration attribute can also be used in embedded Request Timing sub-elements: 
+`<servletTiming/>` or `<jdbcTiming/>`, as follows:
+
+[source, xml]
+----
+<requestTiming includeContextInfo="true" slowRequestThreshold="120s" hungRequestThreshold="10s" sampleRate="1">
+    <servletTiming appName="MyApp" servletName="MyServletApp" slowRequestThreshold="100s" hungRequestThreshold="5s" enableThreadDumps="false"/>
+</requestTiming>`
+----
+
+NOTE: An embedded `<servletTiming/>` or `<jdbcTiming/>` configuration in the server.xml file overrides the configured slow and hung request threshold that are defined in `<requestTiming/>`.  
+
+For more information on the Request Timing feature, please refer to the following documentations:
+- link:{url-prefix}/docs/latest/reference/feature/requestTiming-1.0.html[Open Liberty Documentation on requestTiming-1.0 feature]
+- link:{url-prefix}/docs/latest/reference/config/requestTiming.html[Open Liberty Documentation on requestTiming Configuration]
+
+[#JDBC]
+== Automatic Cleanup of Leaked Connections
+
+Liberty connection management is enhanced with the ability to automatically detect and close unsharable connections that are left open by the application across the end of a request.
+
+Occasionally, application code might forget to close an unsharable connection that it obtains, which prevents the connection from being returned to the connection pool for use by other requests. Over time, these leaked connections can degrade performance and eventually exhaust the connection pool. Liberty connection management now has the ability to detect and automatically close these leaked connections to prevent this from happening.
+
+To take advantage of this new capability, configure one of the Liberty features that leverages the `connectionManager` element. For example, JDBC:
+[source, xml]
+----
+<featureManager>
+  <feature>jdbc-4.2</feature>
+  <feature>jndi-1.0</feature>
+  <!-- more features -->
+</featureManager>
+----
+
+Configure your data sources and connection factories as usual, which automatically leverage the new capability (To disable it, configure `autoCloseConnections="false"` on a `<connectionManager>`).
```

Comment:
lowercase "to":
... capability (to disable it ...

Reply:
Done